serpent/src/lib.rs

```diff
@@ -229,6 +227,7 @@ impl BlockCipher for Serpent {
         Self::new_varkey(key).unwrap()
     }
 
+    #[inline(always)]
```

Comment:
Did these actually help? IMO every `#[inline]` directive should have a comment explaining what it's doing and why that's better than what the LLVM optimizer did by default